chart/values.yaml

```diff
@@ -328,11 +328,11 @@ components:
       http:
         expose: true
         containerPort: 80
-        nodePort: 31080
+        # nodePort: 31080
       https:
         expose: true
         containerPort: 443
-        nodePort: 31443
+        # nodePort: 31443
```

Comment:
why remove?

Reply:
If needs to be removed, can't we just erase the line? :grimacing: